pkgs/by-name/lc/lc0/package.nix

```
@@ -0,0 +1,69 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, python3
+, cmake
+, zlib
+, gtest
+, eigen
+, ...
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lc0";
+  version = "0.30.0";
+
+  src = fetchFromGitHub {
+    owner = "LeelaChessZero";
+    repo = "lc0";
+    rev = "refs/tags/v${version}";
+    sha256 = "sha256-Q85hXNSexHhk6W48tgJLk0Sf32xBipfg2P2SH1FF89Q=";
+    fetchSubmodules = true;
+  };
+
+  patchPhase = ''
+    patchShebangs --build /build/source/scripts/*
+  '';
+
+  nativeBuildInputs = [
+    meson
+    cmake
+    ninja
+    pkg-config
+    python3
+    zlib
+    gtest
+  ];
```

```suggestion
  strictDeps = true;

  nativeBuildInputs = [
    meson
    ninja
    pkg-config
    python3
  ];
  
  buildInputs = [
    eigen
    gtest
    zlib
  ];
```

see https://nixos.org/manual/nixpkgs/unstable/#ssec-stdenv-dependencies
doesn't seem like cmake is needed either